website/docs/tutorials/07-testing/02-cheatcodes/example.md

```diff
@@ -0,0 +1,44 @@
+# `example`
+
+```python
+def example(self, *args: Any, **kwargs: Any) -> None:
```

Comment:
I wonder if `parametrize` wouldn't be a better name as we are already inspired by pytest heavily

Reply:
@mkaput What do you think? You're the original author of the issue for this feature.
`parametrize` would come from pytest
`example` from hypothesis

Reply:
Pytest's `parameterize` has different API than ours. Their approach is more like `examples` (notice plural form), when you declare multiple input sets at once. Here we define inputs one by one hence I think `example` is better suit. 

also `parameters` doesn't sound as nice as `example` imho here.

Reply:
Ok, let's leave example then